dll/win32/uxtheme/nonclient.c

```diff
@@ -1167,19 +1167,10 @@ DrawWindowForNCPreview(
     }
     pcontext->wi.rcClient = rcClientNew;
 
-    SelectObject(hDC, GetThemeSysColorBrush(pcontext->theme, COLOR_WINDOW));
-    HPEN pen = CreatePen(PS_NULL, 0, RGB(0,0,0));
-    SelectObject(hDC, pen);
-
     OffsetRect(&rcClientNew, -pcontext->wi.rcWindow.left, -pcontext->wi.rcWindow.top);
-    if (!(dwExStyle & WS_EX_DLGMODALFRAME))
-    {
-        rcClientNew.right++;
-        rcClientNew.bottom++;
-    }
-    pcontext->wi.rcClient = rcClientNew;
+    
     if (drawClientAreaColor)
-        Rectangle(hDC, rcClientNew.left, rcClientNew.top, rcClientNew.right, rcClientNew.bottom);
+        FillRect(hDC, &rcClientNew, (HBRUSH)(COLOR_WINDOW + 1));
```

Comment:
Ahh. Now I see why maybe the Rectangle function was originally used :oof: because one should use the theme-specific `COLOR_WINDOW` color, and not the currently active one.
However i think you still can use the FillRect function as follows:
```suggestion
    if (drawClientAreaColor)
    {
        HBRUSH hbrWindow = GetThemeSysColorBrush(pcontext->theme, COLOR_WINDOW);
        FillRect(hDC, &rcClientNew, hbrWindow);
        DeleteObject(hbrWindow);
    }
```